### PR TITLE
Temporary avoid RemoteWriteConfig update on anteater/deu01 and anteater/seu01

### PR DESCRIPTION
**Pull request by marieroque**
towards https://github.com/giantswarm/giantswarm/issues/28239

To be able to make some manual configuration on the RemoteWriteConfig on big WC, we chose to temporary avoid RemoteWriteConfig update on anteater/deu01 and anteater/seu01.

## Checklist

I have:

- [x] Described why this change is being introduced
- [ ] Separated out refactoring/reformatting in a dedicated PR
- [x] Updated changelog in `CHANGELOG.md`

### CODE REVIEW

File: service/controller/resource/monitoring/remotewriteconfig/create.go [hunk 1]
```diff
@@ -37,6 +37,14 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 			return microerror.Mask(err)
 		}
 
+		// /!\ Atlas want to make some manual configuration on RemoteWriteConfig on anteater/deu01 and anteater/seu01
+		// We temporary disable the RemoteWriteConfiguration update for those two WC
+		cluster_name := cluster.GetName()
```
Comment (QuentinBisson) [inline service/controller/resource/monitoring/remotewriteconfig/create.go:42]: I'm pretty sure the queueconfiguration is in the remotewritesecret and not the config

File: service/controller/resource/monitoring/remotewriteconfig/create.go [hunk 2]
```diff
@@ -37,6 +37,14 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
...
+		if cluster_name == "deu01" || cluster_name == "seu01" {
```
Comment (QuentinBisson) [inline service/controller/resource/monitoring/remotewriteconfig/create.go:43]: Also, could you ensure the installation is anteater? and use clusterName, please, this is not Python :p

**Comment by marieroque** [comment]
Need to be tested on:
- [ ] `anteater`
- [ ] another installation to test non regression

**Review by QuentinBisson — APPROVED** [review]
LGTM when tested

**Pull request merged**